docker-compose-cam.yml

```diff
@@ -0,0 +1,70 @@
+version: '3.7'
+services:
+  camera-simulator:
+    container_name: camera-simulator
+    image: aler9/rtsp-simple-server
+    ports:
+      - "127.0.0.1:8554:8554"
+  camera-simulator0:
+    build:
+      context: .
+    image: openvino/ubuntu20_data_runtime:2021.4.2
+    container_name: camera-simulator0
+    network_mode: "host"
+    entrypoint: ffmpeg
+    command: "
```

Comment:
Probably this can be refactored in the future by using only 1 camera simulator container and have a for loop on the sample videos and repeat the code.